docs/book/codegen.md

```diff
@@ -87,3 +87,13 @@ return [
     ],
 ];
 ```
+
+## Logging
+
+The `InjectorGenerator` implements the PSR-3 [`LoggerAwareInterface`](http://www.php-fig.org/psr/psr-3/#4-psrlogloggerawareinterface).
```

Comment:
Since the logger can be passed in at instantiation time, and the `*Aware*` usage in ZF was removed for good and well discussed reasons, this is a clear mistake